utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/ParsedSchemaSupport.scala

```diff
@@ -156,7 +158,22 @@ object JsonSchemaSupport extends ParsedSchemaSupport[OpenAPIJsonSchema] {
   override def formValueEncoder(schema: ParsedSchema, mode: ValidationMode): Any => AnyRef = {
     val encoder   = new ToJsonSchemaBasedEncoder(mode)
     val rawSchema = schema.cast().rawSchema()
-    (value: Any) => encoder.encodeOrError(value, rawSchema)
+    (value: Any) => {
+      // In ad-hoc test without schema we create object `{ "Value" = userInputInAdHoc }`, so if present we should just take the input
+      Try {
+        val temp = value.asInstanceOf[Map[String, Map[String, Any]]].head
+        val key  = temp._1
+        // Any try to create a variable with value temp._2 fails
+        if (key.equals("Value")) {
+          temp._2
+        } else Failure
```

Comment:
hm. returning Failure here is intentional?
If is see it right then If `key` is not equal to `Value` we will try to encode Succees(Failure()) in line 174.